src/main/java/it/pagopa/pn/commons/configs/TaxIdInWhiteListParameterConsumer.java

```diff
@@ -0,0 +1,49 @@
+package it.pagopa.pn.commons.configs;
+
+import it.pagopa.pn.commons.abstractions.ParameterConsumer;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+import static it.pagopa.pn.commons.utils.LogUtils.maskTaxId;
+
+@Slf4j
+@Component
```

Comment:
Siccome facciamo l'attivazione lato microservizio, possiamo eliminare il @Component. Inoltre potremmo creare in pn-common una classe ValidationCFConfiguration dove vengono creati i @Bean di ValidationUtils e TaxIdInWhiteListParameterConsumer (vedi RestTemplateFactory). A quel punto i microservizi utilizzeranno un solo attivatore invece che due (estendendo la classe Configuration).